core/config/project_settings.cpp

```diff
@@ -493,10 +493,16 @@ Error ProjectSettings::_setup(const String &p_path, const String &p_main_pack, b
 		}
 	}
 
-	// Attempt with a user-defined main pack first
+#if defined(PCK_ENCRYPTION_ENABLED) && !defined(TOOLS_ENABLED)
+	// PCK encryption is enabled, load only from encrypted pack
```

Comment:
```suggestion
	// PCK encryption is enabled, load only from encrypted pack.
```
Comment style nitpick, same elsewhere